aries-site/src/components/content/CollapsibleSection.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Collapsible, Text } from 'grommet';
+import { FormUp, FormDown } from 'grommet-icons';
+
+export const CollapsibleSection = ({ label, onClick, ...rest }) => {
+  const [open, setOpen] = useState(false);
+  return (
+    <>
+      <Box
+        background="background-contrast"
+        direction="row"
+        gap="small"
+        onClick={() => {
+          setOpen(!open);
+          if (onClick) onClick();
+        }}
+        margin={{ bottom: 'xxsmall' }}
```

Comment:
```suggestion
        border={open && { side: 'bottom', color: 'border' }}
```
Matches design a bit closer

Reply:
Nice, I added this into my PR.